dojo/models.py

```diff
@@ -1255,7 +1255,6 @@ class Finding(models.Model):
     duplicate_finding = models.ForeignKey('self', editable=False, null=True,
                                           related_name='original_finding',
                                           blank=True, on_delete=models.CASCADE)
-    duplicate_list = models.ManyToManyField("self", editable=False, blank=True)
```

Comment:
@ptrovatelli unsure if this was used before at all?

Reply:
i didn't use it specifically but I think it's displayed in the gui isn't it?

Reply:
it's displayed yes. but the list can already be retrieved via the duplicate_finding ForeignKey so there's no need for an additional ManyToMany relationship. At least that's the hypothesis of this PR. If 1.6RC is already branched of we work/test with this in `dev` for a bit until the next release. (with some support from the PR author)

Reply:
It was used in some views. 
Also in the models.py the function duplicate_finding_set used this duplicate_list before.
However you dont need it if you use the backwards relation

Reply:
In the old situation you could have a finding being a duplicate of multiple findings (via duplicate_list):

```
A->B--|
|     v
->C---D
```

In the new situation this would always have to be a straight line:

```
A->B->C->D
or
A->C->B->D
```

Reply:
We should avoid lists. If I am right, in the new situation it looks like this:
A->D
B->D
C->D

All list will be resolved. Not by the model, but during the deduplication.

Reply:
Yeah that's the other part of this PR I think that a duplicate can never be the original of another issue.